sdks/java/harness/src/main/java/org/apache/beam/fn/harness/state/MultimapUserState.java

```diff
@@ -106,6 +108,23 @@ public MultimapUserState(
         .setWindow(stateKey.getMultimapKeysUserState().getWindow())
         .setKey(stateKey.getMultimapKeysUserState().getKey());
     this.userStateRequest = userStateRequestBuilder.build();
+
+    StateRequest.Builder entriesStateRequestBuilder = StateRequest.newBuilder();
```

Comment:
Am I correct that this object is constructed fresh each time we have a ProcessBundle request for a dofn that uses multimap state? If so we might want to optimize by moving more of these to be lazily initialized when used.

could just leave an optimization comment for now.

Reply:
Good idea. I added a note (we could do the same with the keys StateRequest as well.